### PR TITLE
Yurii Pash

### PR DESCRIPTION
**Pull request by yuriipash**

### CODE REVIEW

File: ypash/HW1/1.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+user_name = input("What is your name?")
```
Comment (kolyasalubov) [inline ypash/HW1/1.py:1]: good job

File: ypash/HW1/2.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+a = int(input('a ='))
```
Comment (kolyasalubov) [inline ypash/HW1/2.py:1]: good job

File: ypash/HW3/1.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                    Unless explicitly silenced.\
+                    In the face of ambiguity, refuse the temptation to guess.\
+                    There should be one-- and preferably only one --obvious way to do it.\
+                    Although that way may not be obvious at first unless you're Dutch.\
+                    Now is better than never.\
+                    Although never is often better than *right* now.\
+                    If the implementation is hard to explain, it's a bad idea.\
+                    If the implementation is easy to explain, it may be a good idea.\
+                    Namespaces are one honking great idea -- let's do more of those!")
+
```
Comment (kolyasalubov) [inline ypash/HW3/1.py:23]: good job

File: ypash/HW3/2.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+digits = 1234
```
Comment (kolyasalubov) [inline ypash/HW3/2.py:1]: good job

File: ypash/HW3/3.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+first_digit = 34
```
Comment (kolyasalubov) [inline ypash/HW3/3.py:1]: good job

File: ypash/HW4/1.py [hunk 6]
```diff
@@ -0,0 +1,8 @@
+digit = int(input())
```
Comment (kolyasalubov) [inline ypash/HW4/1.py:1]: good job

File: ypash/HW7/1/1.py [hunk 7]
```diff
@@ -0,0 +1,8 @@
+def get_max_num(num_1, num_2):
```
Comment (kolyasalubov) [inline ypash/HW7/1/1.py:1]: good job

File: ypash/HW7/2/modul.py [hunk 8]
```diff
@@ -0,0 +1,13 @@
+def area_of_rectangle(a, b):
```
Comment (kolyasalubov) [inline ypash/HW7/2/modul.py:1]: good job

File: ypash/HW7/3/3.py [hunk 9]
```diff
@@ -0,0 +1,15 @@
+def letter_caunter(word):
```
Comment (kolyasalubov) [inline ypash/HW7/3/3.py:1]: good job

File: ypash/HW8/Task_1/main.py [hunk 10]
```diff
@@ -0,0 +1,4 @@
+from models import *
```
Comment (kolyasalubov) [inline ypash/HW8/Task_1/main.py:1]: good job

File: ypash/HW8/Task_2/2.py [hunk 11]
```diff
@@ -0,0 +1,20 @@
+import re
```
Comment (kolyasalubov) [inline ypash/HW8/Task_2/2.py:1]: good job

File: ypash/HW8/Task_3/1.py [hunk 12]
```diff
@@ -0,0 +1,10 @@
+from math import pow
```
Comment (kolyasalubov) [inline ypash/HW8/Task_3/1.py:1]: good job

File: ypash/HW8/Task_3/func.py [hunk 13]
```diff
@@ -0,0 +1,16 @@
+def area_rectangle(a, b):
```
Comment (kolyasalubov) [inline ypash/HW8/Task_3/func.py:1]: good job

File: ypash/HW8/my_pyOWM.py [hunk 14]
```diff
@@ -0,0 +1,28 @@
+from pyowm import OWM
```
Comment (kolyasalubov) [inline ypash/HW8/my_pyOWM.py:1]: good job

File: ypash/HW9/1/1.py [hunk 15]
```diff
@@ -0,0 +1,25 @@
+from random import randint
```
Comment (kolyasalubov) [inline ypash/HW9/1/1.py:1]: good job

File: ypash/HW9/2/Rect_pygame.py [hunk 16]
```diff
@@ -0,0 +1,57 @@
+import pygame
```
Comment (kolyasalubov) [inline ypash/HW9/2/Rect_pygame.py:1]: good job

File: ypash/HW1/1.py (hunk 1 above)
Comment (yuriipash) [reply ypash/HW1/1.py:1]: +

File: ypash/HW1/2.py (hunk 2 above)
Comment (yuriipash) [reply ypash/HW1/2.py:1]: Thank you)

File: ypash/HW3/1.py (hunk 3 above)
Comment (yuriipash) [reply ypash/HW3/1.py:23]: Thank you)

File: ypash/HW3/2.py (hunk 4 above)
Comment (yuriipash) [reply ypash/HW3/2.py:1]: Thank you)

File: ypash/HW3/3.py (hunk 5 above)
Comment (yuriipash) [reply ypash/HW3/3.py:1]: Thank you)

File: ypash/HW8/Task_3/1.py (hunk 12 above)
Comment (yuriipash) [reply ypash/HW8/Task_3/1.py:1]: Thank you)

File: ypash/HW8/Task_3/func.py (hunk 13 above)
Comment (yuriipash) [reply ypash/HW8/Task_3/func.py:1]: Thank you)

File: ypash/HW8/my_pyOWM.py (hunk 14 above)
Comment (yuriipash) [reply ypash/HW8/my_pyOWM.py:1]: Thank you)

File: ypash/HW9/1/1.py (hunk 15 above)
Comment (yuriipash) [reply ypash/HW9/1/1.py:1]: Thank you)

File: ypash/HW9/2/Rect_pygame.py (hunk 16 above)
Comment (yuriipash) [reply ypash/HW9/2/Rect_pygame.py:1]: Thank you)

File: ypash/HW4/1.py (hunk 6 above)
Comment (yuriipash) [reply ypash/HW4/1.py:1]: Thank you)

File: ypash/HW7/1/1.py (hunk 7 above)
Comment (yuriipash) [reply ypash/HW7/1/1.py:1]: Thank you)

File: ypash/HW7/2/modul.py (hunk 8 above)
Comment (yuriipash) [reply ypash/HW7/2/modul.py:1]: Thank you)

File: ypash/HW7/3/3.py (hunk 9 above)
Comment (yuriipash) [reply ypash/HW7/3/3.py:1]: Thank you)

File: ypash/HW8/Task_2/2.py (hunk 11 above)
Comment (yuriipash) [reply ypash/HW8/Task_2/2.py:1]: Thank you)

File: ypash/HW8/Task_1/main.py (hunk 10 above)
Comment (yuriipash) [reply ypash/HW8/Task_1/main.py:1]: Thank you)

**Pull request merged**